src/ucp/wireup/select.c

```diff
@@ -205,6 +212,33 @@ static int ucp_wireup_check_amo_flags(const uct_tl_resource_desc_t *resource,
     return 0;
 }
 
+/**
+ * Calulates a small value to overcome float imprecision between two scores
+ */
+static double ucp_wireup_calc_score_epsilon(double score1, double score2)
+{
+    return (score1 + score2) * (1e-6);
+}
+
+/**
+ * Check if the one score is bettter than the another one
+ */
+static int ucp_wireup_score_better(double score1, double score2)
```

Comment:
maybe have a function like ucp_wireup_score_cmp(a,b) which will return -1/0/1 if less,eq,more respectively?

Reply:
yes, it is better - done